src/main/java/com/aws/greengrass/secretmanager/crypto/RSAMasterKey.java

```diff
@@ -14,18 +17,34 @@
 import java.security.PrivateKey;
 import java.security.PublicKey;
 
+import static com.aws.greengrass.secretmanager.SecretManagerService.SECRET_MANAGER_SERVICE_NAME;
+
 public class RSAMasterKey implements MasterKey {
+    private static final Logger logger = LogManager.getLogger(RSAMasterKey.class);
     // Metadata used by JCEMasterKey, so that another instance of JCEMasterKey cannot be used
     // to encrypt/decrypt payload even if it uses the same key.
     private static final String KEY_PROVIDER = "gg:secrets";
-    private static final String WRAPPING_ALGO = "RSA/ECB/OAEPWithSHA-256AndMGF1Padding";
+    private static final String[] WRAPPING_ALGOS = {"RSA/ECB/OAEPWithSHA-256AndMGF1Padding", "RSA/ECB/PKCS1Padding"};
```

Comment:
It should not matter for secrets which block format we use, but I would be careful about choosing one here. There are 2 considerations, 1. need to be careful if either of them is considered insecure. Just use PKCS1Padding if that works. How did you validate that PKCS1 works? 2. It should be backward compatible, customers can get into situation if they migrate their file based key into a TPM, previous encrypted secrets wont be able to decrypt with new block scheme.

Reply:
It cannot be compatible that's just not possible since our interface is only supporting a single provider. The secrets need to be resynced in that case. I'll try to figure out how to make sure it syncs.

I can't really test until the pkcs plugin has been deployed to maven.

Reply:
PKCS1 is 1 of only 2 algorithms offered by SunPKCS11: `RSA/ECB/PKCS1Padding` and `RSA/ECB/NoPadding` from the source code of SunPKCS11

Reply:
I'm ok to just switch to only PKCS1, that's not a problem